catalog/spatial/registry/registry-identification-plugin/src/main/java/org/codice/ddf/registry/identification/IdentificationPlugin.java

```diff
@@ -61,14 +82,26 @@
 
     private Parser parser;
 
-    private FederationAdminService federationAdmin;
+    private CatalogFramework catalogFramework;
+
+    private FilterBuilder filterBuilder;
 
     private ParserConfigurator marshalConfigurator;
 
     private ParserConfigurator unmarshalConfigurator;
 
     private Set<String> registryIds = ConcurrentHashMap.newKeySet();
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(IdentificationPlugin.class);
```

Comment:
I don't see any of these private static variables being used. Were you planning on adding them in a different PR or can they be removed?